app/mailers/student_mailer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+#
+# Mailer class for user e-mails
+class StudentMailer < ApplicationMailer
+  default from: 'no-reply@vesta.app' # SHOULD WE HAVE AN ADMIN ADDRESS?
```

Comment:
we should probably have an admin address; it will probably make sense for students to want to respond to emails individually in some circumstances.